sql/src/main/java/io/crate/analyze/relations/RelationAnalyzer.java

```diff
@@ -128,6 +128,8 @@ protected AnalyzedRelation visitQuery(Query node, StatementAnalysisContext conte
             analysisContext.addSourceRelation(twoRelationsUnion.first().getQualifiedName().toString(), twoRelationsUnion.first());
             SelectAnalysis selectAnalysis = new SelectAnalysis(analysisContext);
             for (int i = 0; i < twoRelationsUnion.fields().size(); i++) {
+                // Add the output symbols (column references) instead of the fields
+                // to be able to convert them to InputColumns in the planning phase
```

Comment:
fields are usually converted to references in the normlization phase. Why isn't that working in this case?